controllers/MarketplaceController.php

```diff
@@ -2,19 +2,45 @@
 
 namespace humhub\modules\xcoin\controllers;
 
-use Yii;
 use humhub\components\Controller;
-use humhub\modules\xcoin\models\Account;
 use humhub\modules\space\widgets\Image as SpaceImage;
 use humhub\modules\xcoin\models\Asset;
-use humhub\modules\xcoin\models\Exchange;
+use humhub\modules\xcoin\models\Product;
+use Yii;
 
 class MarketplaceController extends Controller
 {
 
     public function actionIndex()
     {
-        return $this->render('index');
+        $products = Product::find()->where(['status' => Product::STATUS_AVAILABLE ])->all();
```

Comment:
at some point we might need some pagination. 
but not now, we first need to get products.

Reply:
Yes i confirm , i'll optimize that and add pagination once we have a lot of products :+1: